webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/index.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { Grid, Flex, FlexItem, Button } from '@patternfly/react-core';
+import { registerCoreCards } from './CardRegistry';
+import Slot from '../../../common/Slot';
+import { STATUS } from '../../../../constants';
+import '../Overview/styles.css';
+import { translate as __ } from '../../../../common/I18n';
+
+const DetailsTab = ({ response, status, hostName }) => {
+  useEffect(() => {
+    //  This is a workaround for adding gray background inspiring pf4 desgin
+    //  TODO: delete it when pf4 layout (Page copmponent) is implemented in foreman
```

Comment:
Is there some issue tracking this? Linking to a Redmine makes it easier to see when it can be addressed.
```suggestion
    //  TODO: delete it when pf4 layout (Page component) is implemented in foreman
```